frontend/src/components/Listing/listing.jsx

```diff
@@ -2,62 +2,273 @@ import React, { Component } from "react";
+    // .then(response =>{
+    //   this.setState({
+    //     imageSource: response.data[0].source
+    //   })
+    // })
+
+    // this.setState({
+    //   imageSource: this.getEmergencyFoundImg(apartment1)
+    // })
+    axios.get("http://localhost:3005/listingArray")
+    .then(response =>{
+      this.props.setListingArray(response.data)
+    })
   }
 
+  componentWillUnmount = () =>{
+    this.props.setListingArray([])
+    this.props.setPageNum(1);
+  }
+
+  getEmergencyFoundImg = (urlImg) => {
+    var img = new Image();
+    img.src = urlImg;
+    img.crossOrigin = 'Anonymous';
+  
+    var canvas = document.createElement('canvas'),
+      ctx = canvas.getContext('2d');
+  
+    canvas.height = img.naturalHeight;
+    canvas.width = img.naturalWidth;
+    ctx.drawImage(img, 0, 0);
+  
+    var b64 = canvas.toDataURL('image/png').replace(/^data:image.+;base64,/, '');
+    return b64;
+  };
+
   render() {
     return (
-      <div
-        className="listing"
-      >
-        {/* {this.props.washRoom} */}
-        <img
-          style = {{
-            width: "100%",
-            height: "100%"
-          }}
-          src = {apartment1}
-          alt="apartment 1"
-        />
+      <div className = "listingContent">
+        <div className="leftPanel">
+          {/* {this.props.listingArray.map(listingDetail, index) => (
+
+          )} */}
+          {this.props.listingArray && this.props.listingArray.length !== 0 
+          ? 
+            this.props.listingArray
+            .slice((this.props.pageNum - 1) * 10, this.props.pageNum * 10)
+            .map((listingDetail, index)=>{
+            // return
+            const currIndex = this.props.pageNum * 10 + index;
+            return (
+              <Paper
+                key = {currIndex}
+                elevation = {3}
+                style = {{
+                  width: "100%",
+                  minWidth: 600,
+                  height: 150,
+                  marginTop: 16,
+                  marginBottom: 16
+                }}
+                className = "individualListingContent"
+              >
+                {/* This is for the image area */}
+                <span className = "listingImageArea">
+                  <img
+                    style = {{
+                      width: "100%",
+                      height: "100%"
+                    }}
+                    // src={`data:image/png;base64,${this.state.imageSource}`}
+                    src={apartment1}
+                    alt="apartment"
+                  />
+                </span>
+
+                <div className = "listingTextAndIcon">
+                  <span className = "listingHeader">
+                  
+                    {/* This is for the listing title area */}
+                    <span className = "listingTitle">
+                      {listingDetail.title}
+                    </span>
+
+                    <Typography
+                      type="title"
```

Comment:
Some element attributes in this file have spaces around the `=` and some do not. I think it could be good to get ESLint and have it run on save since that would help make this consistent.

Reply:
I don't think the spaces around the attributes would make any difference